build/common.sh

```diff
@@ -199,8 +199,13 @@ function kube::build::prepare_docker_machine() {
   }
   docker-machine start "${DOCKER_MACHINE_NAME}" > /dev/null
   # it takes `docker-machine env` a few seconds to work if the machine was just started
-  while ! docker-machine env ${DOCKER_MACHINE_NAME} &> /dev/null; do
-    sleep 1
+  while ! docker_machine_out=$(docker-machine env ${DOCKER_MACHINE_NAME} 2>&1); do
+    if [[ ${docker_machine_out} == *"Error checking TLS connection"* ]]; then
+      echo ${docker_machine_out}
+      docker-machine regenerate-certs ${DOCKER_MACHINE_NAME}
```

Comment:
`docker-machine` has a prompt. There is no prompt in the `common.sh` script itself.